pkg/streamformatter/streamformatter_test.go

```diff
@@ -84,9 +85,18 @@ func TestJSONFormatProgress(t *testing.T) {
 	if msg.Status != "action" {
 		t.Fatalf("Status must be 'action', got: %s", msg.Status)
 	}
-	if msg.ProgressMessage != progress.String() {
-		t.Fatalf("ProgressMessage must be %s, got: %s", progress.String(), msg.ProgressMessage)
+
+	// The progress will always be in the format of:
+	// [=========================>                         ]     15 B/30 B 404933h7m11s
+	// The last entry '404933h7m11s' is the timeLeftBox.
+	// However, the timeLeftBox field may change as progress.String() depends on time.Now().
+	// Therefore, we have to strip the timeLeftBox from the strings to do the comparison.
+
+	// Compare the progress strings before the timeLeftBox
+	if !strings.HasPrefix(msg.ProgressMessage, "[=========================>                         ]     15 B/30 B") {
```

Comment:
You could use a variable for the expected string to not write it twice :angel: :

```go
expectedProgress := "[=========================>                         ]     15 B/30 B"
```